<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dadosCadastrais</class>
 <widget class="QWidget" name="dadosCadastrais">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dados Cadastrais - [GeriFÃ¡cil] </string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>130</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Cidade:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>170</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Telefone:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_voltar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>60</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Voltar</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="in_telefone">
   <property name="geometry">
    <rect>
     <x>527</x>
     <y>170</y>
     <width>400</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>443</x>
     <y>80</y>
     <width>51</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Cliente:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="in_nome">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>80</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_cidade">
   <property name="geometry">
    <rect>
     <x>527</x>
     <y>130</y>
     <width>400</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="b_salvar">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>240</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Salvar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
